lib/modules/shipments_api_retriever.rb

```diff
@@ -3,22 +3,51 @@ module ShipmentsApiRetriever
   AVAILABLE_TYPES = ['category', 'importing', 'exporting', 'commodity', 'species', 'taxonomy']
 
   def self.api_call(params)
-    response = HTTParty.get(Rails.application.secrets['species_api_url'],
+    response = HTTParty.get(Rails.application.secrets['species_api_url'] + '/shipments',
```

Comment:
I think the `shipments` bit should be already in the URL in the secrets.
Same thing applies below

Reply:
Ah, ingore my comment. You did like so to use also the other API endpoints I believe.
So that's fine, we just need to remember to update the .env file